pages/[wallet]/index.tsx

```diff
@@ -252,26 +253,28 @@ export default function Dashboard() {
             </Flex>
           )}
           {projects && projects.length ? (
-            <Tabs>
-              <TabList>
-                <Tab>V2 Contracts</Tab>
-                <Tab>V1 Projects</Tab>
-              </TabList>
-              <TabPanels>
-                <TabPanel px={0} pt={8}>
-                  <ContractTable combinedList={combinedList} />
-                </TabPanel>
-                <TabPanel px={0} pt={8}>
-                  <OldProjects projects={projects} />
-                </TabPanel>
-              </TabPanels>
-            </Tabs>
+            <>
+              <Tabs>
+                <TabList>
+                  <Tab>V2 Contracts</Tab>
+                  <Tab>V1 Projects</Tab>
+                </TabList>
+                <TabPanels>
+                  <TabPanel px={0} pt={8}>
+                    <ContractTable combinedList={combinedList} />
+                  </TabPanel>
+                  <TabPanel px={0} pt={8}>
+                    <OldProjects projects={projects} />
+                  </TabPanel>
+                </TabPanels>
+              </Tabs>
+              <LearnMoreSection />
```

Comment:
so we're not showing `<LearnMoreSection />` anymore **unless* you have v1 projects?

Reply:
Fixed ^